benefits/eligibility/templates/eligibility/unverified--agency-card.html

```diff
@@ -0,0 +1,15 @@
+{% extends "eligibility/unverified.html" %}
+{% load i18n %}
+
+{% block unverified-headline %}
+    {% translate "Your card information may not have been entered correctly." %}
+{% endblock unverified-headline %}
+
+{% block unverified-body %}
+    {% translate "The number and last name must be entered exactly as they appear on your MST Courtesy Card. Please check your card and try again, or contact your transit agency for help." %}
```

Comment:
Need to either use a variable here for the agency card name or have a template per agency card flow.

Reply:
Decided to have a template per agency card flow. This is more in line with our current pattern where we are ok with some duplication